Src/Web/Web.Shared.Net.Tests/HandlerTelemetryProcessorTests.cs

```diff
@@ -103,5 +103,31 @@ public void ProcessReturnsIfSuccessfulRequestHandlerIsFiltered()
 
             Assert.AreEqual(0, spy.ReceivedCalls);
         }
+
+        [TestMethod]
```

Comment:
Looks like this class needs cleanup method that would set HttpContext.Current to null. And in general in other places we try to encapsulate HttpContext.Current (see ResolvePlatformContext) so that 1. in tests we do not need to care about singletons 2. if in future we want to pass context from somewhere else we do not need to search for it in all places...